satpy/tests/test_writers.py

```diff
@@ -253,13 +256,15 @@ def test_enhance_with_sensor_entry2(self):
 
 
 class TestYAMLFiles(unittest.TestCase):
+
```

Comment:
I'm 99% sure this is an undesired non-PEP8 change (blank line between class declaration and class docstring).

Reply:
Hmm. Must be something `autopep8` did, as I didn't touch the existing tests. Removed the blank lines.